packages/prismarine/src/network/packet/CreativeContentPacket.ts

```diff
@@ -9,11 +9,16 @@ export default class CreativeContentPacket extends DataPacket {
     public entries: any[] = [];
 
     public encodePayload() {
+        this.writeUnsignedVarInt(0);
+        /* Disabled for now cause it is not tested
```

Comment:
Is there something particularly wrong with this?

Reply:
well ive not tested it thats why i disabled it, last time before i make changes in that it was causing issues.

Reply:
I think the best path here would either be to instead of commenting out this add a `TODO` comment and/or create a new PR that fixes this based on this PR.